cluster-autoscaler/Dockerfile.amd64

```diff
@@ -11,9 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-ARG BASEIMAGE=gcr.io/distroless/static:nonroot-amd64
-FROM $BASEIMAGE
+FROM golang:1.23 AS builder
```

Comment:
You'll want to change this to:

`FROM --platform=$BUILDPLATFORM golang:1.23 as builder`

to support cross-platform compilation (see https://docs.docker.com/build/building/multi-platform/#cross-compilation)

Reply:
done